core/scripts/vrfv2plus/testnet/util.go

```diff
@@ -0,0 +1,305 @@
+		e.Owner,
+		e.Ec,
+		common.HexToAddress(bhsAddress))
+	helpers.PanicErr(err)
+	coordinatorAddress = helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID)
+
+	// Set LINK and LINK ETH
+	coordinator, err := vrf_coordinator_v2plus.NewVRFCoordinatorV2Plus(coordinatorAddress, e.Ec)
+	helpers.PanicErr(err)
+
+	linkTx, err := coordinator.SetLINK(e.Owner, common.HexToAddress(linkAddress))
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, linkTx, e.ChainID)
+
+	linkETHTx, err := coordinator.SetLinkEthFeed(e.Owner, common.HexToAddress(linkEthAddress))
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, linkETHTx, e.ChainID)
+	return coordinatorAddress
+}
+
+func deployBatchCoordinatorV2(e helpers.Environment, coordinatorAddress common.Address) (batchCoordinatorAddress common.Address) {
+	_, tx, _, err := batch_vrf_coordinator_v2plus.DeployBatchVRFCoordinatorV2Plus(e.Owner, e.Ec, coordinatorAddress)
+	helpers.PanicErr(err)
+	return helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func eoaAddConsumerToSub(e helpers.Environment,
+	coordinator vrf_coordinator_v2plus.VRFCoordinatorV2Plus, subID uint64, consumerAddress string) {
+	txadd, err := coordinator.AddConsumer(e.Owner, subID, common.HexToAddress(consumerAddress))
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, txadd, e.ChainID)
+}
+
+func eoaCreateSub(e helpers.Environment, coordinator vrf_coordinator_v2plus.VRFCoordinatorV2Plus) {
+	tx, err := coordinator.CreateSubscription(e.Owner)
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func eoaDeployConsumer(e helpers.Environment,
+	coordinatorAddress string,
+	linkAddress string,
+	keyHash common.Hash,
+	nativePayment bool) (
+	consumerAddress common.Address,
+	consumer *vrf_v2plus_single_consumer.VRFV2PlusSingleConsumerExample) {
+	_, tx, consumer, err := vrf_v2plus_single_consumer.DeployVRFV2PlusSingleConsumerExample(
+		e.Owner,
+		e.Ec,
+		common.HexToAddress(coordinatorAddress),
+		common.HexToAddress(linkAddress),
+		uint32(1000000), // gas callback
+		uint16(5),       // confs
+		uint32(1),       // words
+		keyHash,
+		nativePayment)
+	helpers.PanicErr(err)
+	return helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID), consumer
+}
+
+func eoaFundSubscription(e helpers.Environment,
+	coordinator vrf_coordinator_v2plus.VRFCoordinatorV2Plus, linkAddress string, amount *big.Int, subID uint64) {
+	linkToken, err := link_token_interface.NewLinkToken(common.HexToAddress(linkAddress), e.Ec)
+	helpers.PanicErr(err)
+	bal, err := linkToken.BalanceOf(nil, e.Owner.From)
+	helpers.PanicErr(err)
+	fmt.Println("Initial account balance:", bal, e.Owner.From.String(), "Funding amount:", amount.String())
+	b, err := utils.ABIEncode(`[{"type":"uint64"}]`, subID)
+	helpers.PanicErr(err)
+	e.Owner.GasLimit = 500000
```

Comment:
Don't believe this is needed.

Reply:
Removed this line `e.Owner.GasLimit = 500000` and tested without it